cadc-util/src/main/java/ca/nrc/cadc/net/HttpPost.java

```diff
@@ -401,9 +401,12 @@ private void doPost(HttpURLConnection conn, Map<String,List<Object>> params, Map
                 } else {
                     sb.append(pe.getKey());
                     sb.append("=");
+                    if (v instanceof char[]) {
+                        v = String.valueOf((char[]) v);
+                    }
                     sb.append(URLEncoder.encode(v.toString(), "UTF-8"));
-                    sb.append("&");
                 }
+                sb.append("&");
```

Comment:
Is this okay here considering the LINE_FEED adds in the 'if' part of this stanza?

Reply:
good catch... fixed. It used to be inside the else